### PR TITLE
Updated editor signing procedure for windows

### PR DESCRIPTION
**Pull request by JCash**

### CODE REVIEW

File: editor/scripts/bundle.py [hunk 1]
```diff
@@ -153,19 +153,31 @@ def sign_files(platform, options, dir):
...
 
+        if not os.path.exists(certificate):
+            print("Certificate file does not exist:", certificate)
+            sys.exit(1)
+
+        certificate_pass = 'invalid'
+        with open(certificate_pass_path, 'rb') as f:
+            certificate_pass = f.read()
+
+        signtool = os.path.join(os.environ['DYNAMO_HOME'], 'ext','SDKs','Win32','WindowsKits','10','bin','10.0.18362.0','x64','signtool.exe')
```
Comment (JCash) [inline editor/scripts/bundle.py:169]: Not sure how to find the tool on the system, so I used our packaged sdk instead.

File: editor/scripts/bundle.py [hunk 2]
```diff
@@ -153,19 +153,31 @@ def sign_files(platform, options, dir):
...
         exec_command([
-            'SignTool',
+            signtool,
             'sign',
             '/fd', 'sha256',
             '/a',
             '/f', certificate,
             '/p', certificate_pass,
-            '/tr', 'http://timestamp.comodoca.com',
+            '/tr', 'http://timestamp.digicert.com',
```
Comment (JCash) [inline editor/scripts/bundle.py:180]: Neither comodoca nor the sectigo servers worked 🤔

File: editor/scripts/bundle.py [hunk 3]
```diff
@@ -396,7 +408,7 @@ def sign(options):
             sign_files('darwin', options, os.path.join(jdk_path, "lib", "jspawnhelper"))
             sign_files('darwin', options, os.path.join(sign_dir, "Defold.app"))
         elif 'win32' in platform:
-            sign_files('win32', options,  os.path.join(sign_dir, "Defold.exe"))
+            sign_files('win32', options, os.path.join(sign_dir, "Defold", "Defold.exe"))
```
Comment (JCash) [inline editor/scripts/bundle.py:411]: Nor sure when this folder was added...

File: editor/scripts/bundle.py (hunk 1 above)
Comment (britzl) [reply editor/scripts/bundle.py:169]: I assumed it was installed and on the path on GitHub CI Windows machines?

**Review by britzl — APPROVED** [review]
Great stuff!

**Pull request merged**